Makefile

```diff
@@ -208,6 +208,3 @@ all: $(ALL_UCODES)
 clean:
 	@printf "$(WARNING)Deleting all built microcode files$(NO_COL)\n"
 	@rm -rf $(ALL_OUTPUT_DIRS)
-
-doc:
-  doxygen Doxyfile
```

Comment:
why you remove that ?

Reply:
it caused compilation errors when trying to compile f3dex3

Reply:
it's because it's two space and not a true tabulation